src/main/java/com/devcv/review/application/ReviewServiceImpl.java

```diff
@@ -125,8 +128,8 @@ public Review register(Long resumeId, Long memberId, ReviewDto resumeReviewDto)
         }
         Member member = memberRepository.findById(memberId)
                 .orElseThrow(() -> new MemberNotFoundException(ErrorCode.MEMBER_NOT_FOUND));
-        Order order = orderIdOpt.get();
-        Long orderId = orderIdOpt.get().getOrderId();
+        Order order = orderResumes.get(0).getOrder();
+        Long orderId = order.getOrderId();
         resumeReviewDto.setResumeId(resumeId);
         resumeReviewDto.setMemberId(memberId);
         resumeReviewDto.setOrderId(orderId);
```

Comment:
```suggestion
        Order order = orderResumes.get(0).getOrder();
        String orderNumber = order.getOrder().getOrderNumber();
        resumeReviewDto.setResumeId(resumeId);
        resumeReviewDto.setMemberId(memberId);
        resumeReviewDto.setOrderNumber(orderNumber);
```

클라이언트에서 주문내역을 조회하는 방법은 `orderNumber`를 이용한 `주문상세조회`와 memberId를 이용한 `주문내역리스트조회`만 가능하기 때문에 `orderId`보단 `orderNumber`를 내려주는게 좋지 않을까요?? 👀

Reply:
음...!! 저는 클라이언트의 구매후기 등록 시 주문 이력 확인을 위해서라면 더 정확한  order 내 pk를 이용해 참조하는게 더 정확할 것 같다고 생각했었는데 클라이언트의 입장이라면 그럴 수도 있겠네요...!!🙌🙌  리팩토링 pr때 참고하여 반영하겠습니다!! 감사합니다👍